hooks/useNotification.tsx

```diff
@@ -3,20 +3,41 @@ import { notificationState } from 'lib/recoil-store/atoms'
 import { generateRandomId } from 'lib/utils'
 import { useRecoilState } from 'recoil'
 import { NOTIFICATION_TIMEOUT_DEFAULT } from 'lib/constants'
+import { MutableRefObject } from 'react'
 
-const useNotification = () => {
+const styles = Object.freeze({
+    opacity: '0',
+    transform: 'translateY(-50%)',
```

Comment:
I think without the vertical translate it would look a bit better, wdyt?

Reply:
If you want a translate perhaps we could make it horizontal, from right as fadeIn and to right as fadeOut

Reply:
I would also make the animation faster